gateway-manager/data/services.json

```diff
@@ -0,0 +1,16 @@
+{
+  "demo-service": {
+    "key": "demo",
+    "name": "<b>Demo</b> Service",
+    "icon": "${host}/${public_realm}/demo-service/assets/logo.png",
+    "link": "demo-service/web-app",
+    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+  },
+  "%kibana%": {
```

Comment:
I added the `%` on purpose to avoid being detected in real deployments, those are only examples of entries.